### DISCUSSION TITLE
I Want To Contribute

### DISCUSSION THREAD

**Discussion by aayushchugh** [obsproject/obs-studio#4956] [Development Questions] [answered]
Hi. I have found a problem on the obs website in https://obsproject.com/welcome this page so I want to contribute in that website but I can't find repo of that so can anyone can help me with that

**Comment by WizardCM** [comment]
What issue did you find?

**Reply by aayushchugh** [reply]
when you download obs from the home page when you get redirected to https://obsproject.com/welcome
than the favicon does not load you have to reload the page to see that 

I can't see that issue today so I think someone already solved that

**Comment by dodgepong** [comment, marked as answer]
The website does not have a repo. Please report the issue and we can fix it.
